### PR TITLE
Fix multiple critical issues in README.md

### PR DESCRIPTION
**Pull request by hatarist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,15 +83,15 @@ console.log(cryptoSocket.Exchanges['bitfinex'])
 
 ## FAQ
 
-** Does this store data? **
+### Does this store data?
 
 Nope. Ticker values (in most cases last sale price) is stored in memory.
```
Comment (BEaStia) [inline README.md]: Are stored

**Review by BEaStia — CHANGES_REQUESTED** [review]

**Review by BEaStia — APPROVED** [review]

**Comment by BEaStia** [comment]
you're pidor, but it's okay if it works!

**Comment by redcap3000** [comment]
I will require at least 5 other reviewers before I can confidently review the changes requested. It would also help if these reviewers had at least starred this repo. Thank you.

**Review by faustinoaq — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -49,12 +57,12 @@ These exchanges should support all markets that they have, and will be simple to
 
 **Note**
 
-As of now **Poloniex** exchange only has one open socket that sends back all data. The above syntax is not recommeneded unless you are only following one symbol; as it will open up multiple sockets that return all data, and filter out your selections.
+As of now, **Poloniex** exchange only has one open socket that sends back all data. The above syntax is not recommeneded unless you are only following one symbol, as it will open up multiple sockets that returns all data, and filter out your selections.
```
Comment (QuickBASIC) [inline README.md:60]: s/`returns`/`will return`

File: README.md [hunk 3]
```diff
@@ -49,12 +57,12 @@ These exchanges should support all markets that they have, and will be simple to
...
 __________
 
 ### echoExchange()
 
-A simple printout of all open ticker quotes.
+A simple printout of all opened ticker quotes.
```
Comment (QuickBASIC) [inline README.md:65]: s/`opened`/`open`

Changing this doesn't add any clarity; "open ticker quotes" are ticker quotes that are open (referring to at the current time) not ticker quotes that were at some point in the past opened.

File: README.md [hunk 4]
```diff
@@ -83,15 +91,15 @@ console.log(cryptoSocket.Exchanges['bitfinex'])
...
-Nope. Ticker values (in most cases last sale price) is stored in memory.
+Nope. Ticker values (in most cases last sale price) are stored in memory.
 
-*** Why? ***
+### Why?
 
 Other popular BTC average modules require incredible amount of bandwidth at regular intervals which can make deployment difficult in many situations. Websockets are obviously the way to go for real time data, as constantly querying many exchanges can be taxing.
 
-This module **does not include exchange API's that do not have web sockets**. Why? Theres a billion other modules that do something similar, and perhaps this is a wake up call to any exchange that does not offer rhobust websocket support.
+This module **does not include exchange API's that do not have web sockets**. Why? There's a billion other modules that do something similar, and perhaps this is a wake up call to any exchange that does not offer robust websocket support.
```
Comment (QuickBASIC) [inline README.md:102]: s/`There's`/`There are`

We're referring to multiple items (billions of modules).

**Review by QuickBASIC — CHANGES_REQUESTED** [review]

**Review by Dobermensch — CHANGES_REQUESTED** [review]
Make changes as requested in comments

**Comment by redcap3000** [comment]
Well well. Russian Trolls at their finest. If anyone wants to continue to nitpick my README I'm more than open to changes.

**Review by redcap3000 — APPROVED** [review]
redcap3000 approves these changes.

**Comment by redcap3000** [comment]
So anyone here know javascript? Maybe we can correct the actual code or someone can write some much needed unit tests. As robotic as I am; I am unable to evaluate myself in a programmatic fashion. Thank you!

**Pull request merged**